tests/integration/test_backup_restore_on_cluster/test_cancel_backup.py

```diff
@@ -644,6 +644,7 @@ def test_long_disconnection_stops_backup():
             "CANNOT_READ_ALL_DATA",
             "NETWORK_ERROR",
             "TABLE_IS_READ_ONLY",
+            "NO_REPLICA_HAS_PART",
```

Comment:
This test disconnects ClickHouse from Keeper, so it's expected to get error `No active replica has part` during this test.